sdk/nodejs/applicationloadbalancing/listener.ts

```diff
@@ -33,17 +33,17 @@ export class Listener extends pulumi.CustomResource {
     /**
      * An Action block. Action blocks are documented below.
      */
-    public readonly defaultActions: pulumi.Output<{ targetGroupArn: string, type: string }[]>;
+    public readonly defaultAction: pulumi.Output<{ fixedResponse?: { contentType: string, messageBody?: string, statusCode: string }, redirect?: { host?: string, path?: string, port?: string, protocol?: string, query?: string, statusCode: string }, targetGroupArn?: string, type: string }>;
```

Comment:
Interesting.  This will be a breaking change.  Looks like a case there `MaxItems==1` was added in TF, which is not really breaking, but that causes us to project this differently.  

I don't think there is anything we can/should do here - but worth marking this as breaking and adding a note to the PR.